lib/event_sourcery/aggregate_root.rb

```diff
@@ -1,5 +1,55 @@
 module EventSourcery
+  #
+  # EventSourcery::AggregateRoot provides a foundation for writing your own AggregateRoot classes.
```

Comment:
Maybe "... for writing your own aggregate root classes" since "Aggregate root" here is the name of a pattern?